monkestation/code/modules/clothing/head/misc.dm

```diff
@@ -113,6 +113,15 @@
 	user.update_inv_head() //Regenerate the wearer's head appearance so that they have real-time hat updates.
 
 
+/obj/item/clothing/head/cowboy_discount
```

Comment:
Wrong location for this.
I'd suggest checking the file structure for similar hats.